diff -up VTK-9.1.0/GUISupport/Qt/QVTKOpenGLWindow.cxx.omv~ VTK-9.1.0/GUISupport/Qt/QVTKOpenGLWindow.cxx
--- VTK-9.1.0/GUISupport/Qt/QVTKOpenGLWindow.cxx.omv~	2022-03-29 01:07:14.391756488 +0200
+++ VTK-9.1.0/GUISupport/Qt/QVTKOpenGLWindow.cxx	2022-03-29 01:06:56.796632303 +0200
@@ -259,6 +259,7 @@ void QVTKOpenGLWindow::paintGL()
 
     const QSize deviceSize = this->size() * this->devicePixelRatioF();
     const auto fmt = this->context()->format();
+#ifndef GL_ES_VERSION_2_0
     if (fmt.stereo() && this->RenderWindow->GetStereoRender() &&
       this->RenderWindow->GetStereoType() == VTK_STEREO_CRYSTAL_EYES)
     {
@@ -268,6 +269,7 @@ void QVTKOpenGLWindow::paintGL()
         this->defaultFramebufferObject(), GL_BACK_RIGHT, QRect(QPoint(0, 0), deviceSize));
     }
     else
+#endif
     {
       this->RenderWindowAdapter->blit(
         this->defaultFramebufferObject(), GL_BACK_LEFT, QRect(QPoint(0, 0), deviceSize));
