diff -up VTK-9.1.0/Rendering/OpenGL2/vtkOpenGLRenderWindow.cxx.omv~ VTK-9.1.0/Rendering/OpenGL2/vtkOpenGLRenderWindow.cxx
--- VTK-9.1.0/Rendering/OpenGL2/vtkOpenGLRenderWindow.cxx.omv~	2022-03-29 01:11:09.476340243 +0200
+++ VTK-9.1.0/Rendering/OpenGL2/vtkOpenGLRenderWindow.cxx	2022-03-29 01:10:49.661211399 +0200
@@ -586,6 +586,10 @@ bool vtkOpenGLRenderWindow::GetUsingSRGB
 #ifdef GL_DRAW_BUFFER
     glGetIntegerv(GL_DRAW_BUFFER, &attachment);
 #endif
+#ifdef GL_ES_VERSION_2_0
+    // GLES only has the GL_BACK color
+    // attachment for the default framebuffer
+#else
     // GL seems odd with its handling of left/right.
     // if it says we are using GL_FRONT or GL_BACK
     // then convert those to GL_FRONT_LEFT and
@@ -598,6 +602,7 @@ bool vtkOpenGLRenderWindow::GetUsingSRGB
       // the setting the user requested
       return this->UseSRGBColorSpace;
     }
+#endif
     if (attachment == GL_BACK)
     {
       attachment = GL_BACK_LEFT;
