--- VTK/Utilities/vtknetcdf/CMakeLists.txt.orig	2011-11-11 19:09:31.972095937 -0200
+++ VTK/Utilities/vtknetcdf/CMakeLists.txt	2011-11-11 19:09:35.489111959 -0200
@@ -298,6 +298,7 @@ ADD_DEFINITIONS("-DNC_DLL_EXPORT")
 # Apply user-defined properties to the library target.
 IF(VTK_LIBRARY_PROPERTIES)
   SET_TARGET_PROPERTIES(vtkNetCDF PROPERTIES ${VTK_LIBRARY_PROPERTIES})
+  SET_TARGET_PROPERTIES(vtkNetCDF_cxx PROPERTIES ${VTK_LIBRARY_PROPERTIES})
 ENDIF(VTK_LIBRARY_PROPERTIES)
 
 #-----------------------------------------------------------------------------
