diff -up VTK-8.2.0/IO/Movie/CMakeLists.txt.omv~ VTK-8.2.0/IO/Movie/CMakeLists.txt
--- VTK-8.2.0/IO/Movie/CMakeLists.txt.omv~	2019-12-30 03:59:57.237303354 +0100
+++ VTK-8.2.0/IO/Movie/CMakeLists.txt	2019-12-30 04:00:14.129302248 +0100
@@ -30,6 +30,8 @@ set(vtkIOMovie_HDRS
 
 vtk_module_library(vtkIOMovie ${Module_SRCS})
 
+vtk_module_link_libraries(vtkIOMovie LINK_PRIVATE ogg)
+
 if(WIN32 AND VTK_USE_VIDEO_FOR_WINDOWS)
   vtk_module_link_libraries(vtkIOMovie LINK_PRIVATE vfw32)
 endif()
diff -up VTK-8.2.0/ThirdParty/exodusII/vtkexodusII/CMakeLists.txt.omv~ VTK-8.2.0/ThirdParty/exodusII/vtkexodusII/CMakeLists.txt
--- VTK-8.2.0/ThirdParty/exodusII/vtkexodusII/CMakeLists.txt.omv~	2019-12-30 03:48:07.716349815 +0100
+++ VTK-8.2.0/ThirdParty/exodusII/vtkexodusII/CMakeLists.txt	2019-12-30 03:48:19.387349051 +0100
@@ -298,7 +298,7 @@ set(headers
   "${CMAKE_CURRENT_BINARY_DIR}/include/exodus_config.h")
 
 vtk_add_library(vtkexodusII ${sources} ${headers})
-target_link_libraries(vtkexodusII PUBLIC ${vtknetcdf_LIBRARIES})
+target_link_libraries(vtkexodusII PUBLIC ${vtknetcdf_LIBRARIES} -pthread)
 if (NOT VTK_INSTALL_NO_DEVELOPMENT)
   install(FILES
     ${headers}
diff -up VTK-8.2.0/ThirdParty/libharu/vtklibharu/src/CMakeLists.txt.omv~ VTK-8.2.0/ThirdParty/libharu/vtklibharu/src/CMakeLists.txt
--- VTK-8.2.0/ThirdParty/libharu/vtklibharu/src/CMakeLists.txt.omv~	2019-12-30 03:51:56.076334862 +0100
+++ VTK-8.2.0/ThirdParty/libharu/vtklibharu/src/CMakeLists.txt	2019-12-30 03:52:05.571334240 +0100
@@ -102,7 +102,7 @@ if(LIBHPDF_SHARED)
 endif(LIBHPDF_SHARED)
 else ()
   vtk_add_library(vtklibharu ${LIBHPDF_SRCS})
-  target_link_libraries(vtklibharu PRIVATE ${vtkzlib_LIBRARIES} ${vtkpng_LIBRARIES})
+  target_link_libraries(vtklibharu PRIVATE ${vtkzlib_LIBRARIES} ${vtkpng_LIBRARIES} m)
   if (WIN32)
     set_target_properties(vtklibharu
       PROPERTIES
diff -up VTK-8.2.0/Wrapping/Java/CMakeLists.txt.omv~ VTK-8.2.0/Wrapping/Java/CMakeLists.txt
--- VTK-8.2.0/Wrapping/Java/CMakeLists.txt.omv~	2019-12-30 04:06:28.501277734 +0100
+++ VTK-8.2.0/Wrapping/Java/CMakeLists.txt	2019-12-30 04:06:56.777275882 +0100
@@ -339,9 +339,9 @@ if(JOGL_GLUE)
 endif()
 
 # Set the javac source version
-set(VTK_JAVA_SOURCE_VERSION "1.6" CACHE STRING "javac source version")
+set(VTK_JAVA_SOURCE_VERSION "13" CACHE STRING "javac source version")
 mark_as_advanced(VTK_JAVA_SOURCE_VERSION)
-set(VTK_JAVA_TARGET_VERSION "1.6" CACHE STRING "javac target version")
+set(VTK_JAVA_TARGET_VERSION "13" CACHE STRING "javac target version")
 mark_as_advanced(VTK_JAVA_TARGET_VERSION)
 
 # On machines with long paths to VTK (or windows where the command line length
