diff -up VTK-9.3.0.rc1/Wrapping/Java/vtk/sample/rendering/annotation/LabeledCubeAxesActor.java.omv~ VTK-9.3.0.rc1/Wrapping/Java/vtk/sample/rendering/annotation/LabeledCubeAxesActor.java
--- VTK-9.3.0.rc1/Wrapping/Java/vtk/sample/rendering/annotation/LabeledCubeAxesActor.java.omv~	2023-08-15 22:23:57.309970713 +0200
+++ VTK-9.3.0.rc1/Wrapping/Java/vtk/sample/rendering/annotation/LabeledCubeAxesActor.java	2023-08-15 22:24:13.166089412 +0200
@@ -66,10 +66,10 @@ public class LabeledCubeAxesActor {
     cubeAxesActor.SetCamera(renderer.GetActiveCamera());
     cubeAxesActor.SetBounds(points.GetBounds());
     cubeAxesActor.SetXTitle("Date");
-    cubeAxesActor.SetXAxisMinorTickVisibility(0);
+    cubeAxesActor.SetXAxisMinorTickVisibility(false);
     cubeAxesActor.SetAxisLabels(0, getXLabels());
     cubeAxesActor.SetYTitle("Place");
-    cubeAxesActor.SetYAxisMinorTickVisibility(0);
+    cubeAxesActor.SetYAxisMinorTickVisibility(false);
     cubeAxesActor.SetAxisLabels(1, getYLabels());
     cubeAxesActor.SetZTitle("Value");
     renderer.AddActor(cubeAxesActor);
