diff -up VTK-9.0.0/Common/Python/CMakeLists.txt.omv~ VTK-9.0.0/Common/Python/CMakeLists.txt
--- VTK-9.0.0/Common/Python/CMakeLists.txt.omv~	2020-05-21 21:48:15.131466870 +0200
+++ VTK-9.0.0/Common/Python/CMakeLists.txt	2020-05-21 21:48:28.771646780 +0200
@@ -3,3 +3,4 @@ set(classes
 
 vtk_module_add_module(VTK::CommonPython
   CLASSES ${classes})
+vtk_module_link(VTK::CommonPython PRIVATE python3.8)
diff -up VTK-9.0.0/Filters/Python/CMakeLists.txt.omv~ VTK-9.0.0/Filters/Python/CMakeLists.txt
--- VTK-9.0.0/Filters/Python/CMakeLists.txt.omv~	2020-05-21 21:46:29.279070803 +0200
+++ VTK-9.0.0/Filters/Python/CMakeLists.txt	2020-05-21 21:47:20.947752230 +0200
@@ -3,3 +3,4 @@ set(classes
 
 vtk_module_add_module(VTK::FiltersPython
   CLASSES ${classes})
+vtk_module_link(VTK::FiltersPython PRIVATE python3.8)
diff -up VTK-9.0.0/Rendering/PythonContext2D/CMakeLists.txt.omv~ VTK-9.0.0/Rendering/PythonContext2D/CMakeLists.txt
--- VTK-9.0.0/Rendering/PythonContext2D/CMakeLists.txt.omv~	2020-05-21 21:40:24.447260306 +0200
+++ VTK-9.0.0/Rendering/PythonContext2D/CMakeLists.txt	2020-05-21 21:40:37.108427204 +0200
@@ -3,3 +3,4 @@ set(classes
 
 vtk_module_add_module(VTK::PythonContext2D
   CLASSES ${classes})
+vtk_module_link(VTK::PythonContext2D PRIVATE python3.8)
diff -up VTK-9.0.0/ThirdParty/hdf5/vtkhdf5/src/CMakeLists.txt.omv~ VTK-9.0.0/ThirdParty/hdf5/vtkhdf5/src/CMakeLists.txt
--- VTK-9.0.0/ThirdParty/hdf5/vtkhdf5/src/CMakeLists.txt.omv~	2020-05-21 21:44:54.596822153 +0200
+++ VTK-9.0.0/ThirdParty/hdf5/vtkhdf5/src/CMakeLists.txt	2020-05-21 21:55:30.236852503 +0200
@@ -1127,6 +1127,7 @@ set (gen_SRCS ${HDF5_GENERATED_SOURCE_DI
 add_custom_target (gen_${HDF5_LIB_TARGET} ALL DEPENDS ${HDF5_GENERATED_SOURCE_DIR}/gen_SRCS.stamp1 ${HDF5_GENERATED_SOURCE_DIR}/gen_SRCS.stamp2)
 
 add_library(vtkhdf5_src ${common_SRCS} ${gen_SRCS} ${H5_PUBLIC_HEADERS} ${H5_PRIVATE_HEADERS} ${H5_GENERATED_HEADERS})
+target_link_libraries(vtkhdf5_src PRIVATE m dl)
 target_include_directories(vtkhdf5_src
   PRIVATE
     ${CMAKE_CURRENT_SOURCE_DIR}
diff -up VTK-9.0.0/ThirdParty/libharu/vtklibharu/src/CMakeLists.txt.omv~ VTK-9.0.0/ThirdParty/libharu/vtklibharu/src/CMakeLists.txt
--- VTK-9.0.0/ThirdParty/libharu/vtklibharu/src/CMakeLists.txt.omv~	2020-05-21 21:37:17.796800395 +0200
+++ VTK-9.0.0/ThirdParty/libharu/vtklibharu/src/CMakeLists.txt	2020-05-21 21:37:38.153068627 +0200
@@ -108,6 +108,7 @@ else ()
       PROPERTIES
         DEFINE_SYMBOL HPDF_DLL_MAKE)
   endif ()
+  vtk_module_link(VTK::libharu PRIVATE m)
   target_include_directories(libharu
     PUBLIC
       "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../include>"
diff -up VTK-9.0.0/ThirdParty/libxml2/vtklibxml2/CMakeLists.txt.omv~ VTK-9.0.0/ThirdParty/libxml2/vtklibxml2/CMakeLists.txt
--- VTK-9.0.0/ThirdParty/libxml2/vtklibxml2/CMakeLists.txt.omv~	2020-05-21 21:36:23.910090410 +0200
+++ VTK-9.0.0/ThirdParty/libxml2/vtklibxml2/CMakeLists.txt	2020-05-21 21:39:09.817276621 +0200
@@ -256,6 +256,7 @@ target_compile_definitions(libxml2
   PRIVATE
     _GNU_SOURCE
     _REENTRANT)
+vtk_module_link(VTK::libxml2 PRIVATE m pthread dl)
 
 if(NOT BUILD_SHARED_LIBS)
   target_compile_definitions(libxml2
diff -up VTK-9.0.0/ThirdParty/sqlite/vtksqlite/CMakeLists.txt.omv~ VTK-9.0.0/ThirdParty/sqlite/vtksqlite/CMakeLists.txt
--- VTK-9.0.0/ThirdParty/sqlite/vtksqlite/CMakeLists.txt.omv~	2020-05-21 21:33:32.493832768 +0200
+++ VTK-9.0.0/ThirdParty/sqlite/vtksqlite/CMakeLists.txt	2020-05-21 21:41:34.222180119 +0200
@@ -4,6 +4,7 @@ vtk_module_add_module(VTK::sqlite
           vtk_sqlite_mangle.h
           "${CMAKE_CURRENT_BINARY_DIR}/vtksqlite_export.h"
   HEADERS_SUBDIR "vtksqlite")
+vtk_module_link(VTK::sqlite PRIVATE pthread dl)
 
 target_compile_definitions(sqlite
   PRIVATE
diff -up VTK-9.0.0/ThirdParty/tiff/vtktiff/libtiff/CMakeLists.txt.omv~ VTK-9.0.0/ThirdParty/tiff/vtktiff/libtiff/CMakeLists.txt
--- VTK-9.0.0/ThirdParty/tiff/vtktiff/libtiff/CMakeLists.txt.omv~	2020-05-21 21:23:11.818677456 +0200
+++ VTK-9.0.0/ThirdParty/tiff/vtktiff/libtiff/CMakeLists.txt	2020-05-21 21:29:41.496793142 +0200
@@ -146,6 +146,7 @@ vtk_module_add_module(VTK::tiff
   HEADERS ${tiff_HEADERS}
   HEADERS_SUBDIR "vtktiff/libtiff"
   PRIVATE_HEADERS ${nodist_tiff_HEADERS})
+vtk_module_link(VTK::tiff PRIVATE m)
 endif ()
 if(HAVE_LD_VERSION_SCRIPT)
   set_target_properties(tiff PROPERTIES LINK_FLAGS
diff -up VTK-9.0.0/Utilities/PythonInterpreter/CMakeLists.txt.omv~ VTK-9.0.0/Utilities/PythonInterpreter/CMakeLists.txt
--- VTK-9.0.0/Utilities/PythonInterpreter/CMakeLists.txt.omv~	2020-05-21 21:32:32.777046623 +0200
+++ VTK-9.0.0/Utilities/PythonInterpreter/CMakeLists.txt	2020-05-21 21:32:45.482213872 +0200
@@ -8,3 +8,4 @@ set(headers
 vtk_module_add_module(VTK::PythonInterpreter
   CLASSES ${classes}
   HEADERS ${headers})
+vtk_module_link(VTK::PythonInterpreter PRIVATE python3.8)
diff -up VTK-9.0.0/Wrapping/Python/CMakeLists.txt.omv~ VTK-9.0.0/Wrapping/Python/CMakeLists.txt
diff -up VTK-9.0.0/Wrapping/PythonCore/CMakeLists.txt.omv~ VTK-9.0.0/Wrapping/PythonCore/CMakeLists.txt
--- VTK-9.0.0/Wrapping/PythonCore/CMakeLists.txt.omv~	2020-05-21 21:30:26.679387385 +0200
+++ VTK-9.0.0/Wrapping/PythonCore/CMakeLists.txt	2020-05-21 21:30:41.127577438 +0200
@@ -35,3 +35,4 @@ vtk_module_add_module(VTK::WrappingPytho
   HEADERS ${headers}
   PRIVATE_HEADERS ${private_headers}
   LIBRARY_NAME_SUFFIX "${PYTHON_MAJOR_VERSION}${PYTHON_MINOR_VERSION}")
+vtk_module_link(VTK::WrappingPythonCore PRIVATE python3.8)
diff -up VTK-9.0.0/ThirdParty/exodusII/vtkexodusII/CMakeLists.txt.omv~ VTK-9.0.0/ThirdParty/exodusII/vtkexodusII/CMakeLists.txt
--- VTK-9.0.0/ThirdParty/exodusII/vtkexodusII/CMakeLists.txt.omv~	2020-05-22 00:18:26.338800561 +0200
+++ VTK-9.0.0/ThirdParty/exodusII/vtkexodusII/CMakeLists.txt	2020-05-22 00:18:46.781885781 +0200
@@ -301,6 +301,7 @@ vtk_module_add_module(VTK::exodusII
   SOURCES ${sources}
   HEADERS ${headers}
   HEADERS_SUBDIR "vtkexodusII/include")
+vtk_module_link(VTK::exodusII PRIVATE pthread)
 target_compile_definitions(exodusII
   PRIVATE
     exoIIc_EXPORTS)
diff -up VTK-9.0.0/Web/Core/CMakeLists.txt.omv~ VTK-9.0.0/Web/Core/CMakeLists.txt
--- VTK-9.0.0/Web/Core/CMakeLists.txt.omv~	2020-05-22 14:51:16.672718619 +0200
+++ VTK-9.0.0/Web/Core/CMakeLists.txt	2020-05-22 14:51:38.965950987 +0200
@@ -7,3 +7,4 @@ set(classes
 
 vtk_module_add_module(VTK::WebCore
   CLASSES ${classes})
+vtk_module_link(VTK::WebCore PRIVATE python3.8)
